### PR TITLE
Add interface for adding co owners

### PR DESCRIPTION
**Pull request by nd0761**

### CODE REVIEW

**Review by conradgrobler — APPROVED** [review]

**Review by ioannanedelcu — APPROVED** [review]

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -42,18 +42,6 @@ async function checkLog() {
   return loggedIn;
 }
 
-// Show a marker's settings area.
-function showSettings(contentId) {
```
Comment (nd0761) [inline src/main/webapp/script.js]: You sure this section should be deleted?

Comment (nd0761) [reply src/main/webapp/script.js]: O.... it still working.... magic? oO

Comment (ioannanedelcu) [reply src/main/webapp/script.js]: Yeah, I updated it and added it in the `mapScript.js`

**Comment by conradgrobler** [comment]
Please update the PR title to match the format for commit titles (e.g. reuse the title form #15)

**Pull request merged**